pexv2/pd.go

```diff
@@ -29,13 +29,24 @@ type Constraints struct {
 
 // Field contains the requirements for a given field within a proof
 type Field struct {
-	ID      string   `json:"id,omitempty"`
-	Name    string   `json:"name,omitempty"`
-	Path    []string `json:"path,omitempty"`
-	Purpose string   `json:"purpose,omitempty"`
-	Filter  Filter   `json:"filter,omitempty"`
+	ID        string       `json:"id,omitempty"`
+	Name      string       `json:"name,omitempty"`
+	Path      []string     `json:"path,omitempty"`
+	Purpose   string       `json:"purpose,omitempty"`
+	Filter    *Filter       `json:"filter,omitempty"`
```

Comment:
super nit
```suggestion
	Filter    *Filter      `json:"filter,omitempty"`
```

Reply:
thanks for saving us from future eye twitches